src/index.tsx

```diff
@@ -5,7 +5,6 @@ export {default} from './box'
 export {default as splitProps} from './utils/split-props'
 export {default as splitBoxProps} from './utils/split-box-props'
```

Comment:
It might be interesting to export BoxProps itself as well for interface composition

Reply:
Also named exports would be better (f.e. in split-box-props.ts: `export const splitBoxProps` and in here `export { splitBoxProps}`)

Reply:
I like that! We can do that in a follow up PR.